student.rb

```diff
@@ -0,0 +1,109 @@
+require 'csv'
+class Student
+    attr_accessor :name, :roll_number, :bengali, :english, :maths, :computer, :total, :percentage, :percentile
+    def initialize name, roll, bengali,english,maths,computer,t,per,p
+        @name=name
+        @roll_number = roll
+        @bengali = bengali
+        @english = english
+        @maths = maths
+        @computer = computer
+        @total=t
+        @percentage=per
+        @percentile=p
+    end
+
+    def calculate_total
+        @total = @bengali+@english+@maths+@computer
+    end
+    def calculate_percentage
+        @percentage=(@total/400.0)*100
+        p @percentage
+    end
+    def calculate_percentile
+        @percentile=(@percentage/10).floor
```

Comment:
The percentile formula is wrong.
Please, see below link for percentile example
ref: https://www.cuemath.com/percentile-formula/